test/steps/crypto/1_given.js

```diff
@@ -93,6 +93,12 @@ export function aPassphraseWithPublicKey() {
 	this.test.ctx.publicKey = publicKey;
 }
 
+export function aStringDenotingNumberOfAccounts() {
```

Comment:
`aNumberOfAccounts` seems understandable
and also is this function really relates to `crypto`?

Reply:
Changed this function's name to aNumberOfAccounts.

I made this part of crypto, as the aPassphraseWithPrivateKeyAndPublicKeyAndAddress function was part of crypto file.
I think that options/1_given is more apt for this, shall I move it there?